README.md

```diff
@@ -239,7 +239,7 @@ It is occasionally useful to run locally, or against alternate integration branc
 
 # Android Support
 
-Significant work has been taken to ensure Android compatibility (in particular for `minSdkVersion` 16). Note that the
+Significant work has been taken to ensure Android compatibility (in particular for `minSdkVersion` 26). Note that the
```

Comment:
That's a big jump. What controls it? And what android version does it correspond to?

Reply:
https://www.nutrient.io/blog/java-8-in-android-apps/#how-it-works

"Java 8 has been supported natively since Android SDK 26."

Android 16 I believe was the minimum when we supported Java 7.

Reply:
Even 26 is a long time ago. Seems fine.